h/static/scripts/vendor/jquery.scrollintoview.js

```diff
@@ -109,29 +109,37 @@
 
 				var animOptions = {};
 
+				var yDir = "none";
```

Comment:
> On the other hand, I thought (and still think) that there is a clear use-case for the user wanting to find an annotation inside the document, starting from the annotation card. Do you see a problem with that use-case?

Yes. It inverts the primary-secondary relationship of the document and its annotations. It discourages reading. It confuses two different uses for annotation and thereby muddles our product.